internal/trust/trust_darwin.go

```diff
@@ -53,6 +68,11 @@ func (k keychain) StoreCertificate(file string, i Informer) error {
 }
 
 func (keychain) Instructions() string {
+	return "1. Download the certificate: kubectl get secret kyma-gateway-certs -n istio-system -o jsonpath='{.data.cert}' > kyma.crt\n" +
```

Comment:
This step is for the new installation right? In the old installation the cert is still in a configmap in `kyma-system` right?
From the git history, the old instructions were:
`kubectl get configmap net-global-overrides -n kyma-installer -o jsonpath='{.data.global\\.ingress\\.tlsCrt}' | base64 --decode > kyma.crt`